orttraining/orttraining/test/gradient/gradient_ops_test.cc

```diff
@@ -1552,11 +1613,11 @@ TEST(Synchronization, WaitAndRecordEventMany) {
   const size_t event_count = 16;
   for (int i = 0; i < 8; ++i) {
     std::thread thread_pool[2 * event_count];
-    for (int j = 0; j < event_count; ++j) {
+    for (int j = 0; j < static_cast<int>(event_count); ++j) {
```

Comment:
>int [](start = 9, length = 3)

size_t ?

Reply:
These two places is to remove some warning. Since the following std::thread(wait_event, j); takes j as the 2nd parameter which is int type, so keep j as int in the for loop and cast event_count to int for comparison.